DOC-3356: New feature documentation for `tinymceai`. (#3976)
tinymce/tinymce-docs: modules/ROOT/partials/toolbar-button-ids/tinymceai-toolbar-buttons.adoc

[cols="2,3",options="header"]
|===
|Toolbar button identifier |Description

> |image:icons-premium/ai-assistant.svg[Chat icon,24px] `+tinymceai-chat+` |Opens the AI Chat sidebar for conversations with the AI assistant.

Reviewer: Nitpick: Should we avoid using the term "AI assistant" since that's what the last plugin was called?

```suggestion
|image:icons-premium/ai-assistant.svg[Chat icon,24px] `+tinymceai-chat+` |Opens the AI Chat sidebar for conversations with the AI agent.
```

> |image:icons/ai-prompt.svg[Quick Actions icon,24px] `+tinymceai-quickactions+` |Opens the Quick Actions menu (improve writing, fix grammar, translate, and similar).

Reviewer: ```suggestion
|image:icons/ai-prompt.svg[Quick Actions icon,24px] `+tinymceai-quickactions+` |Opens the AI Quick Actions menu (improve writing, fix grammar, translate, and similar).
```

> |image:icons-premium/ai-review.svg[Review icon,24px] `+tinymceai-review+` |Opens the AI Review sidebar for suggested edits and content review.

Reviewer: ```suggestion
|image:icons-premium/ai-review.svg[Review icon,24px] `+tinymceai-review+` |Opens the AI Review sidebar for running content review workflows.
```
Avoiding the phrase "suggested edits". Not 100% about "workflows" though?
